resources/com.yubico.yubikey_manager.metainfo.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.yubico.yubikey_manager</id>
+  
+  <name>Yubikey manager</name>
+  <summary>  Cross-platform application for configuring any YubiKey over all USB interfaces. </summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Could you explain what this metadata license is?

Reply:
The license of the metainfo file, It is normally CC0 as this file may be combined with other files such as the desktop file and to prevent license incompatibility CC0 is used

Reply:
Thanks!